tests/test_sequence_generator.py

```diff
@@ -113,7 +140,9 @@ def _test_save_and_load(self, scripted_module):
 JIT_MSG = "Targeting OSS scriptability for the 1.6 release"
 
 
-@unittest.skipIf(torch.__version__ < "1.6.0", JIT_MSG)
+@unittest.skipIf(
```

Comment:
You can add the necessary pytorch version to the CI testing to make sure this test works and passes.

Reply:
I can add one nightly version like 1.13.0.dev20220613 as one more test.

Reply:
That sounds good to me